runtime-common/stdlib/math/bcmath-functions.cpp

```diff
@@ -490,22 +490,20 @@ void f$bcscale(int64_t scale) noexcept {
   }
 }
 
-string f$bcdiv(const string& lhs_str, const string& rhs_str, int64_t scale) noexcept {
+string f$bcdiv(const string& lhs_str, const string& rhs_str, Optional<int64_t> scale) noexcept {
   auto& math_lib_context{MathLibContext::get()};
   const auto& math_lib_constants{MathLibConstants::get()};
-  if (scale == std::numeric_limits<int64_t>::min()) {
-    scale = math_lib_context.bc_scale;
-  }
-  if (scale < 0) {
-    php_warning("Wrong parameter scale = %" PRIi64 " in function bcdiv", scale);
-    scale = 0;
+  int64_t scale_val{!scale.has_value() ? math_lib_context.bc_scale : scale.val()};
```

Comment:
Let's make it `const` (and NIT: let's rename scale to opt_scale and scale_val to scale)